File: js/anagram2.js
exports.anagramsFor = function(word, listOfWords) {
    let resArr = [];
    for(let checking of listOfWords)
    if (word.toLowerCase().split('').sort().join('')
     === checking.toLowerCase().split('').sort().join('')){
         resArr.push(checking)
     }
     return resArr;
};

File: python/anagram2.py
from hashlib import new


def anagrams_for(word, list_of_words):
	res_list=[]
	new_word= list(word.lower())
	new_word.sort()
	for checking in list_of_words:
		new_checking= list(checking.lower())
		new_checking.sort()
		if  "".join(new_word)== "".join(new_checking):
			res_list.append(checking)
	return res_list


